lib/rubocop/cop/layout/space_inside_block_braces.rb

```diff
@@ -7,6 +7,15 @@ module Layout
       # them on configuration. For blocks taking parameters, it checks that the
       # left brace has or doesn't have trailing space depending on
       # configuration.
+      #
+      # @example
+      #   # bad
```

Comment:
`# bad` if `EnforcedStyleForEmptyBraces` config is set with `space`, right?

Maybe you enhance it and add also the inverse configuration.

Reply:
For example: https://github.com/bbatsov/rubocop/blob/fe87238e769d00cb52212c640fdfaefc66699a1b/lib/rubocop/cop/style/alias.rb#L10-L29